api/models/media.py

```diff
@@ -10,3 +10,4 @@ class Media(Base):
     media_type = Column(String, nullable=False)
     artist_id = mapped_column(ForeignKey("artists.id"))
     artist = relationship("Artist", back_populates="media", lazy="joined")
+    url = Column(String, nullable=False)
```

Comment:
I don't remember if this was in sqlalchemy or pydantic, but I remember there was a helpful type called `AnyHttpUrl` which basically checks to make sure that a string is an interpretable HTTP(S) link (similar to the `EmailStr` type for valid email addresses...)